CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 #### Loki
 
 ##### Enhancements
+* [8206](https://github.com/grafana/loki/pull/8206) **yashsoni369**: Updating doc removing `querier.query_timeout` to `limits_requests.query_timeout`.
```

Comment:
nit: Docs changes do not require a changelog entry.

Reply:
Noted

Reply:
Updated @chaudum